### ISSUE TITLE
"between" clause not working

### ISSUE THREAD

**Issue by shuantsu** [voku/simple-mysqli#53]
#### What is this feature about (expected vs actual behaviour)?

I want to use BETWEEN on WHERE clause but I'm getting unexpected results. It's inserting parenthesis.
Documentation doesn't give further explanations...

#### How can I reproduce it?

```
print_r($db->select('orc_lancamentos', ['id BETWEEN' => [100, 200]]));
```

#### Does it take minutes, hours or days to fix?
minutes

#### Any additional information?

RESULTANT QUERY:
```
SELECT * FROM `orc_lancamentos` WHERE (  `id` BETWEEN (100 AND 200) 
)
```

**Comment by voku** [comment]
There is a test at least for "NOT BETWEEN" and I am pretty convinced that it passed the test in the past, so maybe something changed? Anyway, we could simply remove the parenthesis in "DB.php":
```php
$_value = '(' . \implode(' AND ', $_value) . ')';
```

Do you want to open a pull request for that? Otherwise I will do that hopefully on the weekend. 

Happy coding. :+1:

**Comment by shuantsu** [comment]
I can wait :)

**Comment by voku** [comment]
fixed in 8.3.1

---

PS: the test were passed because I tested it only in combination with other conditions. :P

ok:
```sql
SELECT *
FROM test_page
WHERE page_id NOT BETWEEN ('99997' AND '99999') AND 1 = 1;
```

error:
```sql
SELECT *
FROM test_page
WHERE page_id NOT BETWEEN ('99997' AND '99999');

**Comment by shuantsu** [comment]
working!
tysm
